Bayesian Regression/README.md

```diff
@@ -0,0 +1,94 @@
+# Bayesian Linear Regression
+## In the normal(frequentist) Linear regression we generally estimate model parameters ( &#x3B8;) by means of maximum likelihood or MAP estimation.Since MLE can lead to severe overfitting,in particular,to small data regime.MAP does not give a good representation of our uncertainitiee.

```

Comment:
I see that you used ## in paragraph many places,you don't need to use that.use ## / # for heading only

Reply:
okay...thanks for the suggestion